patient.py

```diff
@@ -1,7 +1,6 @@
 import barnum
 from util import *
 
-
```

Comment:
The second blank line needs to stay, https://www.python.org/dev/peps/pep-0008/#blank-lines